lang/strings_english.txt

```diff
@@ -264,7 +264,7 @@ $s_view_state_enum_string = '10:public,50:private';
 $s_priority_enum_string = '10:none,20:low,30:normal,40:high,50:urgent,60:immediate';
 $s_severity_enum_string = '10:feature,20:trivial,30:text,40:tweak,50:minor,60:major,70:crash,80:block';
 $s_reproducibility_enum_string = '10:always,30:sometimes,50:random,70:have not tried,90:unable to reproduce,100:N/A';
-$s_status_enum_string = '10:new,20:feedback,30:acknowledged,40:confirmed,50:assigned,80:resolved,90:closed';
+$s_status_enum_string = '10:new,20:feedback,25:external wait,26:internal wait,30:acknowledged,40:confirmed,50:assigned,80:resolved,85:delivered,90:closed';
```

Comment:
Changes to language files seem to be unrelated to this change.  Btw, you should generally be changing custom_strings_inc.php for such customizations rather than changing core language files.